Resources/Prototypes/Diseases/noninfectious.yml

```diff
@@ -66,5 +66,120 @@
     - !type:DiseasePopUp
       probability: 0.03
 
+# Mutation toxin diseases
+- type: disease
+  id: HumanMutation
+  name: disease-proto-muta-human
+  infectious: false
+  cureResist: 0.10
+  stages:
+    - 0
+    - 60
+  effects:
+    - !type:DiseasePolymorph
+      probability: 1
+      polymorphId: HumanMutationMorph
+      polymorphMessage: disease-polymorph
+      polymorphSound:
+        path: /Audio/Voice/Human/malescream_1.ogg
+      stages:
+        - 1
+  cures:
+    - !type:DiseaseJustWaitCure
+      maxLength: 61 # not sure how stages work, maybe this should be 60 or 60.1 or something
```

Comment:
stages determined at what time a effect trigger, with maxLength like this the polymorph effect will trigger and then a second later they will be cured, which i dont know if its intended (same for the rest of diseases)

Reply:
My interpretation of the feature request was that the cure was something that could prevent the mutation toxin from acting, but would not reverse the transformation. Practically speaking, implementing it so that the transformation was reversed upon cure is much more difficult, and outside the bounds of YAML.  The purpose of the disease being cured directly after the transformation is to prevent the effect of polymorphing the diseased person from happening multiple times (the probability of the effect is 100%), but I don't know how often/in what order these checks are made. I'm also not sure if curing it at all is necessary, as polymorphed mobs may not inherit the diseases of the mob they were polymorphed from. This is unnecessary for monkeys, for example, as they have no disease carrier component.